src/libraries/System.Net.Security/tests/StressTests/SslStress/build-local.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
Is this script same (or very similar) as the one in `HttpStress`? Should we somehow share it (in eng dir potentially)?

Reply:
HttpStress doesn't need `Microsoft.AspNetCore.App` so in that case it's much simpler. There is a lot of duplication including msbuild stuff and scripts, but moving these bits to a shared location would make things only harder to maintain. It's already pretty hard to track what lives in `/eng/docker` vs individual stress projects.

If we had a 3rd stress project that would make these bits worth to dedupe.